docs/dataformats/observation_lists.rst

```diff
@@ -22,23 +22,48 @@ This has the advantage that everyone can work with whatever tool they like. Here
 * `csvkit <https://csvkit.readthedocs.org/en/latest/>`_
 * Your own script using e.g the `Python standard library CSV module <http://docs.python.org/2/library/csv.html>`_ or `pandas <http://pandas.pydata.org>`_
 
-A run list must have at least a column called ``Run``::
+A run list must have at least a column called ``OBS_ID``::
  
-   Run
+   OBS_ID
    42
    43
 
-Usually it has many more columns with information about each run::
- 
-   Run,Telescope_Pattern,Date,Duration,GLON,GLAT,Target,Zenith
-   1234,24,2013-03-22 14:32,1832,83.7,-5.2,Crab Nebula,32
+Usually it has many more columns with information about each observation. A list of Gammapy supported columns is::
+
+   OBS_ID: observation ID as an integer (starting at 0 or 1? does it matter?)
+   RA: pointing position right ascension in equatorial (ICRS) coordinates
+   DEC: pointing position declination in equatorial (ICRS) coordinates
+   AZ: average azimuth angle during the observarion
+   ALT: average altitude angle during the observarion
+   MUON_EFFICIENCY: average muon efficiency of the telescopes
+   TIME_START: start time of the observation stored as number of seconds after the time reference in the header
+   TIME_STOP: end time of the observation in the same forma as TIME_START
+   TIME_OBSERVATION: duration of the observation
+   TIME_LIVE: duration of the observation without dead time
+   TRIGGER_RATE: average trigger rate of the system
+   MEAN_TEMPERATURE: average temperature of the atmosphere
+   N_TELS: number of telescopes participating in the observation
+   TEL_LIST: string with a CSV list of telescope IDs participating in the observation
+   QUALITY: data quality; recommended: "spectral" or "detection" (not used by Gammapy at the moment)
+
+Extra user defined columns are allowed; Gammapy will ignore them.
+
+In order for the extra columns to have full meaning a header is needed with at least the following keywords::
+
+   OBSERVATORY_NAME: name of the observatory where the observations were taken. This is important for instance for coordinate transformations between celestial (i.e. RA/dec) and terrestrial (i.e. az/alt) coordinate systems.
+   TIME_REF_MJD_INT: reference time for other times in the list (i.e. TIME_START/TIME_STOP). Integer value in mean julian days.
+   TIME_REF_MJD_FRA: fraction of integer value defined in TIME_REF_MJD_INT.
+
+TODO: change names already defined in event lists (and in `~gammapy.utils.time.time_ref_from_dict`) ``MJDREFI`` and ``MJDREFF``? Or adopt ``TIME_REF_MJD_INT`` and ``TIME_REF_MJD_FRA``?
+
+TODO: should the observation list have already a header? Or should the header values be defined as columns, then interpreted correspondingly when stored into an ObservationTable? (I might be mixing up 2 things: obs lists and obs tables here? or should they have the same format?)
```

Comment:
I'm not sure what you mean here.

When ObservationTable is read from FITS it will have the FITS header in the `.meta` dict and this should be used.
When ObservationTable is read from CSV there will be no header info.
We should not try to store header key / vals in columns.

I realise there's very different things. A "run list" can be just a list of `OBS_ID`s. A data store index table can contain lots of information that will be needed for `findruns`. As I said, I don't have a full concept yet how best to implement classes that cover all the common use cases of things people do with observation lists for Gammapy.
Here my suggestion would be that you try to implement the common use cases as examples in the tests and see how it goes. Probably this will give clues as to which classes need to be added and if any should be removed.